src/components/src/B2cHomePage/b2c.home.page.tsx

```diff
@@ -55,87 +56,109 @@ const B2CHomePage: React.FunctionComponent = () => {
-                {intl.get('learn-more')}
-              </span>
-            </div>
-          </div>
-        </div>
-        <div className="section section-4 container">
-          <div className="sub-section">
-            <ImageContainer className="small-image image-1" fileName={homeEspot3FileName} imgUrl={homeEspot3} />
-            <div className="text-block">
-              <span className="line line-1">
-                {intl.get('home-sub-espot-container3-first-line')}
-              </span>
-              <span className="line line-2">
-                {intl.get('home-sub-espot-container3-second-line')}
-              </span>
+    <div className="home-page-b2c">
+      <section className="main-banner">
+        <ImageContainer className="main-banner-image" fileName={heroBannerFileName} imgUrl={heroBanner} />
+        <div className="main-banner-title-wrap">
+          <div className="container">
+            <h2 className="goods-heading">{intl.get('b2c-main-banner-heading')}</h2>
+            <div className="main-banner-txt">
+              <p className="goods-description">
+                {intl.get('b2c-main-banner-txt')}
+              </p>
+              <div className="btn-wrap">
+                <button type="button" className="ep-btn primary learn-more-btn">{intl.get('learn-more')}</button>
+              </div>
             </div>
-            <ImageContainer className="small-image image-2" fileName={homeEspot3FileName} imgUrl={homeEspot3} />
           </div>
         </div>
-        <div className="section section-5 container">
-          <div className="sub-section">
-            <ImageContainer className="small-image" fileName={homeEspot4FileName} imgUrl={homeEspot4} />
-            <div className="text-block">
-              <span className="line line-1">
-                {intl.get('home-sub-espot-container4-first-line')}
-              </span>
-              <span className="line line-2">
-                {intl.get('home-sub-espot-container4-second-line')}
-              </span>
-            </div>
-          </div>
+      </section>
+
+      <IndiRecommendationsDisplayMain render={['carousel', 'brand']} configuration={Config.indi} />
+
+      <div className="container">
+        <div className="main-goods">
+          <ul className="main-goods__grid">
+            <li className="main-goods__cell">
+              <div className="main-goods-wrap">
+                <div className="goods-info">
+                  <h5 className="goods-title-small">{intl.get('b2c-product1-label')}</h5>
+                  <h3 className="goods-title">{intl.get('b2c-product1-heading')}</h3>
+                  <div className="btn-wrap">
+                    <button type="button" className="ep-btn primary learn-more-btn">{intl.get('learn-more')}</button>
+                  </div>
+                </div>
+                <ImageContainer className="main-goods-image" fileName={heroImagePourCoffeeFileName} imgUrl={heroImagePourCoffee} />
```

Comment:
Check with Bonnie on this one. I'm not sure if this was intended to be a parallax image.